router/router.go

```diff
@@ -1349,17 +1267,36 @@ func (rt *HandleT) getWorkerPartition(userID string) int {
 	return misc.GetHash(userID) % rt.noOfWorkers
 }
 
-func (rt *HandleT) shouldThrottle(destID, userID string, throttledAtTime time.Time) (canBeThrottled bool) {
-	if !rt.throttler.IsEnabled() {
+func (rt *HandleT) shouldThrottle(job *jobsdb.JobT, parameters JobParametersT, throttledUserMap map[string]struct{}) (
+	limited bool,
+) {
+	if rt.throttlerFactory == nil {
```

Comment:
Another reason why we wouldn't want to start without a `throttlerFactory` is the surge of log messages 😄

Reply:
Converting it to `debug` level. It shouldn't happen anyway since I moved the initialization to the app handlers.

Reply:
Then we don't even need this nil check here, correct?